lib/create-http-client.js

```diff
@@ -27,6 +29,10 @@ export default function createHttpClient (axios, options) {
   const defaultConfig = {
     insecure: false,
     retryOnError: true,
+    logEmitter: new EventEmitter(),
+    logHandler: (log) => console.log(log),
+    errorEmitter: new EventEmitter(),
+    errorHandler: (log) => console.error(log),
```

Comment:
1.The name `errorHandler` will give the impression that user can handle errors there.
2. I think we can merge them also, like the event emitters, to `logHandler: (type, log) => console.log(`[${type}], ${log}`)`